improver/wxcode/utilities.py

```diff
@@ -193,9 +228,9 @@ def interrogate_decision_tree(wxtree: str) -> List[str]:
     # CLI help.
     output = []
     for requirement, uniq_thresh in sorted(requirements.items()):
-        (units,) = {u for (_, u) in uniq_thresh}  # enforces same units
-        thresh_str = ", ".join(map(str, sorted(v for (v, _) in uniq_thresh)))
-        output.append("{} ({}): {}".format(requirement, units, thresh_str))
+        (units,) = {u.units for u in uniq_thresh}  # enforces same units
+        thresh_str = ", ".join(map(str, sorted({v.points[0] for v in uniq_thresh})))
+        output.append("- {} ({}): {}".format(requirement, units, thresh_str))
```

Comment:
Is the minus sign that's included at the start of this string just intended for formatting i.e. like bullet points? It might be more obvious not to include it, to avoid confusion about whether it should be interpreted as a minus. Maybe you could just keep the indentation without the `-`, if the concern is how the text is displayed on the command line when using the `--check-tree` option.

Reply:
I've made it into little sun & cloud symbols, which seems appropriate. Will that pass review?
```
Decision tree OK
Required inputs are:
⛅ probability_of_low_and_medium_type_cloud_area_fraction_above_threshold (1): 0.1875, 0.8125
⛅ probability_of_low_type_cloud_area_fraction_above_threshold (1): 0.85
⛅ probability_of_lwe_precipitation_rate_above_threshold (mm hr-1): 0.03, 1.0
⛅ probability_of_lwe_precipitation_rate_in_vicinity_above_threshold (mm hr-1): 0.1, 1.0
⛅ probability_of_lwe_sleetfall_rate_above_threshold (mm hr-1): 0.03, 1.0
⛅ probability_of_lwe_snowfall_rate_above_threshold (mm hr-1): 0.03, 1.0
⛅ probability_of_number_of_lightning_flashes_per_unit_area_in_vicinity_above_threshold (m-2): 0.0
⛅ probability_of_rainfall_rate_above_threshold (mm hr-1): 0.03, 1.0
⛅ probability_of_texture_of_low_and_medium_type_cloud_area_fraction_above_threshold (1): 0.05
⛅ probability_of_visibility_in_air_below_threshold (m): 1000.0, 5000.0

```